tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml

```diff
@@ -50,7 +50,7 @@ jobs:
   pool: 
     vmImage: 'macOS-10.13'
   steps:
-  
+    - template: templates/set-test-data-variables-step.yml
```

Comment:
Do you need to also update the python packaging pipeline ?

Reply:
No I dont... actually now I can undo this change too...

Reply:
It's ok.